ros/src/waypoint_updater/waypoints_helper.py

```diff
@@ -88,27 +88,33 @@ def get_smoothed_out_waypoints(waypoints):
     :return: list of styx_msgs.msg.Waypoint instances
     """
 
-    xs = [waypoint.pose.pose.position.x for waypoint in waypoints]
-    ys = [waypoint.pose.pose.position.y for waypoint in waypoints]
+    new_waypoints = [(waypoint.pose.pose.position.x, waypoint.pose.pose.position.y) for waypoint in waypoints]
```

Comment:
new_waypoints -> better name would be waypoints_xys

Reply:
will change